app/webpacker/stylesheets/application.scss

```diff
@@ -1,2 +1,11 @@
 @import "../../../node_modules/govuk-frontend/all";
+
+// Not in design system:
+$button-colour: #00823b;
+
 @import "cookie-banner";
+@import "phase-tag";
+
+.text-decoration-underline-dotted {
```

Comment:
Do we want this in the main application.scss file or would it be better if it lived somewhere else?

Reply:
I like it here. :)